metrics/platform-metrics-webconsole-plugin/pom.xml

```diff
@@ -97,6 +97,7 @@
                         <Import-Package>
                             javax.servlet;version="[2.5,3)",
                             javax.servlet.http;version="[2.5,3)",
+                            org.apache.felix.webconsole; version="[3.1.0,4.0.0)",
```

Comment:
Probably won't be an issue. I was just curious why the change was needed since the export version didn't change between karaf 2.4.1 and 2.4.3.